### PR TITLE
update gitignore and remove archive and feed

### LINKED ISSUE #32: .gitignore is incomplete

### PR DESCRIPTION
**Pull request by ashleygwilliams**
fixes #32

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,4 @@
 node_modules
 blog/*.html
+blog/*.xml
+blog/articles/
```
Comment (steveklabnik) [inline .gitignore]: why is this change needed? what does it ignore?

Comment (ashleygwilliams) [inline .gitignore]: `/blog/*.html` doesn't grab the html in `/blog/articles`
